web/modules/custom/collection/collection.module

```diff
@@ -362,6 +362,7 @@ function collection_rdf_entity_view(array &$build, EntityInterface $entity, Enti
     $tag = $entity->getEntityTypeId() . ':' . $entity->id();
     // @see collection_og_membership_presave()
     $tags = Cache::buildTags('joinup-membership', [$tag]);
+    $tags = Cache::mergeTags($tags, ['user_list', 'og_membership_list']);
```

Comment:
This causes this cache entry to be invalidated whenever a membership, in any group, is updated. We need to narrow down the invalidation. What is really missing is the invalidation of the cache tag `joinup-membership:group_type:group_id` in a `hook_og_membership_delete()` hook, the same approach done in `collection_og_membership_presave()`.

Reply:
It is not only on delete. When a membership is approved it should also be invalidated.